website/docs/r/batch_account.html.markdown

```diff
@@ -40,6 +40,72 @@ resource "azurerm_batch_account" "test" {
 }
 ```
 
+## Example Usage with User Subscription mode
```

Comment:
rather than adding another example here - could we move this [into the Examples folder](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples) instead?